Parsedown.php

```diff
@@ -1040,26 +1031,11 @@ public function line($text, $non_nestables=array())
                     $Inline['position'] = $markerPosition;
                 }
 
-                # cause the new element to 'inherit' our non nestables, if appropriate
+                # cause the new element to 'inherit' our non nestables
 
-                foreach ($non_nestables as $key => $non_nestable)
+                foreach ($non_nestables as $non_nestable)
                 {
-                    # array syntax, and depth is sufficient to pass on
-                    if (is_array($non_nestable) and isset($non_nestable[1]) and
-                        is_int($non_nestable[1]) and $non_nestable[1] > 1)
-                    {
-                        $Inline['element']['non_nestables'][] = array($non_nestable[0], $non_nestable[1] -1);
-                    }
-                    # array syntax, and depth is indefinite
-                    elseif (is_array($non_nestable) and ! isset($non_nestable[1]))
-                    {
-                         $Inline['element']['non_nestables'][] = array($non_nestable[0]);
-                    }
-                    # string syntax, so depth is indefinite
-                    elseif ( ! is_array($non_nestable))
-                    {
-                        $Inline['element']['non_nestables'][] = $non_nestable;
-                    }
+                    $Inline['element']['non_nestables'][] = $non_nestable;
```

Comment:
@PhrozenByte I did consider that, though there is no guarantee that the `$Inline['element']['non_nestables']` will exist – so the code suffers a little for readability with that added check IMO. Though I appreciate that's a subjective decision, so I'm happy to favour `array_merge` if you disagree.